rpcs3/Emu/Audio/audio_device_listener.cpp

```diff
@@ -18,7 +18,12 @@ audio_device_listener::audio_device_listener()
 	}
```

Comment:
CoCreateInstance will now always fail when you start a game for the first time, because the audio_device_listener is now created before the xaudio backend calls CoInitialize

Reply:
Fixed